### ISSUE TITLE
UNUSED_LOCAL_VARIABLE tag location

### ISSUE THREAD

**Issue by KamasamaK** [cflint/CFLint#637]
Given the following function
```
<cffunction name="foo" hint="Some function hint" returntype="void" output="false">
	<cfargument name="call" required="true" type="string" hint="The value of call">

	<cfset var bar = "">
	<cfhttp url="#variables.host#" port="22" method="GET" timeout="30" result="local.httpResponse">
		<cfhttpparam type="url" name="call" value="#arguments.call#">
		<cfhttpparam type="url" name="bar" value="#local.bar#">
	</cfhttp>
</cffunction>
```

The UNUSED_LOCAL_VARIABLE issue is triggered and the location is reported on the `cfhttp` tag instead of on `httpResponse`.
